### PR TITLE
add references

### PR DESCRIPTION
**Pull request by Lawrencium77**

### CODE REVIEW

File: Fast and Accurate GPU Quantization for Transformers.md [hunk 1]
```diff
@@ -493,19 +484,30 @@ Although FP8 tensor cores have the same theoretical throughput as INT8, changes
...
+2. Mao, Lei. "[Quantization for Neural Networks](https://leimao.github.io/article/Neural-Networks-Quantization/)" (2020).
+3. [PyTorch Docs - Quantization](https://pytorch.org/docs/stable/quantization.html#model-preparation-for-eager-mode-static-quantization).
+4. Gholami, Amir, et al. "[A survey of quantization methods for efficient neural network inference.](https://arxiv.org/pdf/2103.13630.pdf)" _arXiv preprint arXiv:2103.13630_ (2021).
+5. [TensorRT pytorch-quantization](https://github.com/NVIDIA/TensorRT/tree/master/tools/pytorch-quantization).
+6. He, Horace. "[Making Deep Learning Go Brrrr From First Principles](https://horace.io/brrr_intro.html)" (2022).
+7. Xiao, Guangxuan, et al. "[Smoothquant: Accurate and efficient post-training quantization for large language models.](https://arxiv.org/pdf/2211.10438.pdf)" _arXiv preprint arXiv:2211.10438_ (2022).
+
+
+
+#### Part II: Fast GPU Quantization in Practice
```
Comment (david-macleod) [inline Fast and Accurate GPU Quantization for Transformers.md]: If we decide to go for a single bog don't think we need to break up references

File: Fast and Accurate GPU Quantization for Transformers.md [hunk 2]
```diff
@@ -242,19 +239,19 @@ Thankfully, these lower level details are abstracted away by the cuBLASLt  `cub
...
 
-In contrast, fully-fledged inference frameworks such as **TensorRT (TRT)** and **FasterTransformer** do achieve performance gains. They also manage the complexity of fusing quant & dequant nodes with adjacent operators. This is appealing for common Transformer types such as BERT and GPT, for which they have been heavily optimised. However, the rigid assumptions made by these libraries make them less suitable for more unusual architectures. 
+In contrast, fully-fledged inference frameworks such as [TensorRT](https://github.com/NVIDIA/TensorRT) (TRT) and [FasterTransformer](https://github.com/NVIDIA/FasterTransformer) do achieve performance gains. They also manage the complexity of fusing quant & dequant nodes with adjacent operators. This is appealing for common Transformer types such as BERT [10] and GPT [11], for which they have been heavily optimised. However, the rigid assumptions made by these libraries make them less suitable for more unusual architectures.
```
Comment (david-macleod) [inline Fast and Accurate GPU Quantization for Transformers.md]: Not sure we need references for BERT / GPT (as was referencing GPT family)

File: Fast and Accurate GPU Quantization for Transformers.md (hunk 1 above)
Comment (Lawrencium77) [reply Fast and Accurate GPU Quantization for Transformers.md]: Cool, will put into one block

File: Fast and Accurate GPU Quantization for Transformers.md (hunk 2 above)
Comment (Lawrencium77) [reply Fast and Accurate GPU Quantization for Transformers.md]: Done

**Pull request merged**